lib/logflare/sources/source_routing.ex

```diff
@@ -75,13 +63,13 @@ defmodule Logflare.Logs.SourceRouting do
                   apply(Kernel, :==, [le_value, value])
 
                 operator == :string_contains ->
-                  String.contains?(le_value, value)
+                  String.contains?(le_str_value, value)
 
                 operator == := ->
                   apply(Kernel, :==, [le_value, value])
 
                 operator == :"~" ->
-                  apply(Kernel, :=~, [le_value, ~r/#{value}/u])
+                  apply(Kernel, :=~, [le_str_value, ~r/#{value}/u])
```

Comment:
I think that this would be more readable.
```suggestion
                  le_str_value =~ ~r/#{value}/u
```